engine/apps/api/serializers/paging.py

```diff
@@ -1,72 +1,55 @@
+import typing
+
 from django.core.exceptions import ObjectDoesNotExist
 from rest_framework import serializers
 
 from apps.alerts.models import AlertGroup
+from apps.user_management.models import Organization
 from common.api_helpers.custom_fields import TeamPrimaryKeyRelatedField
 from common.api_helpers.utils import CurrentTeamDefault
 
 
-class UserReferenceSerializer(serializers.Serializer):
-    id = serializers.CharField()
-    important = serializers.BooleanField()
-    instance = serializers.HiddenField(default=None)  # set in UserReferenceSerializer.validate
+class SerializerContext(typing.TypedDict):
+    organization: Organization
 
-    def validate(self, attrs):
-        organization = self.context["organization"]
-
-        try:
-            attrs["instance"] = organization.users.get(public_primary_key=attrs["id"])
-        except ObjectDoesNotExist:
-            raise serializers.ValidationError("User {} does not exist".format(attrs["id"]))
-
-        return attrs
 
+class UserReferenceSerializer(serializers.Serializer):
+    context: SerializerContext
 
-class ScheduleReferenceSerializer(serializers.Serializer):
     id = serializers.CharField()
     important = serializers.BooleanField()
-    instance = serializers.HiddenField(default=None)  # set in ScheduleReferenceSerializer.validate
+    instance = serializers.HiddenField(default=None)  # set in UserReferenceSerializer.validate
 
     def validate(self, attrs):
+        id = attrs["id"]
         organization = self.context["organization"]
 
         try:
-            attrs["instance"] = organization.oncall_schedules.get(public_primary_key=attrs["id"])
+            attrs["instance"] = organization.users.get(public_primary_key=id)
         except ObjectDoesNotExist:
-            raise serializers.ValidationError("Schedule {} does not exist".format(attrs["id"]))
+            raise serializers.ValidationError(f"User {id} does not exist")
 
         return attrs
 
 
 class DirectPagingSerializer(serializers.Serializer):
-    users = UserReferenceSerializer(many=True, required=False, default=list)
-    schedules = ScheduleReferenceSerializer(many=True, required=False, default=list)
+    context: SerializerContext
 
-    escalation_chain_id = serializers.CharField(required=False, default=None)
-    escalation_chain = serializers.HiddenField(default=None)  # set in DirectPagingSerializer.validate
```

Comment:
- get rid of `schedules` + `escalation_chain_id` (and by virtue `escalation_chain`), these concepts are no longer relevant
- introduce `team`